src/storage-client/src/controller.rs

```diff
@@ -293,6 +293,9 @@ pub trait StorageController: Debug {
     /// Get the current configuration, including parameters updated with `update_parameters`.
     fn config(&self) -> &StorageConfiguration;
 
+    /// Return the [PersistLocation] used by this controller.
+    fn persist_location(&self) -> PersistLocation;
```

Comment:
Is this used anywhere?

Reply:
good catch! this is from an earlier version when the code had to get it from the controller